src/main/java/com/mastery/java/task/dto/Employee.java

```diff
@@ -0,0 +1,48 @@
+package com.mastery.java.task.dto;
+
+import com.fasterxml.jackson.annotation.JsonIgnore;
+import com.mastery.java.task.dto.validator.IsGender;
+import io.swagger.annotations.ApiModel;
+import io.swagger.annotations.ApiParam;
+import lombok.*;
+import org.springframework.format.annotation.DateTimeFormat;
+
+import javax.persistence.*;
+import javax.validation.constraints.Digits;
+import javax.validation.constraints.NotBlank;
+import javax.validation.constraints.NotNull;
+import javax.validation.constraints.Size;
+import java.text.SimpleDateFormat;
+import java.time.LocalDate;
```

Comment:
please remove unused imports

Reply:
deleted it, i can't get used to the formatting(